.github/CODEOWNERS

```diff
@@ -30,175 +30,257 @@
 /sdk/servicebus/ @jeremymeng @deyaaeldeen @HarshaNalluru
 
 # PRLabel: %Storage
-/sdk/storage/ @EmmaZhu @XiaoningLiu @jeremymeng @vinjiang @jiacfan @ljian3377
+# ServiceLabel: %Storage
+/sdk/storage/ @EmmaZhu @XiaoningLiu @vinjiang @jiacfan @ljian3377
 
 # PRLabel: %Synapse
+# ServiceLabel: %Synapse
+# AzureSdkOwners: @joheredi
 /sdk/synapse/ @joheredi
 
 # PRLabel: %Tables
+# ServiceLabel: %Tables
+# AzureSdkOwners: @joheredi
 /sdk/tables/ @joheredi
 
 # PRLabel: %Purview
+# ServiceLabel: %Purview
+# AzureSdkOwners: @qiaozha
 /sdk/purview/ @qiaozha
 
 # PRLabel: %Farmbeats
+# ServiceLabel: %Farmbeats
+# AzureSdkOwners: @joheredi
 /sdk/agrifood/ @joheredi
 
 # PRLabel: %ConfidentialLedger
+# ServiceLabel: %ConfidentialLedger
+# AzureSdkOwners: @joheredi
 /sdk/confidentialledger/ @joheredi
 
 # PRLabel: %DocumentTranslator
+# ServiceLabel: %DocumentTranslator
+# AzureSdkOwners: @joheredi
 /sdk/documenttranslator/ @joheredi
 
 # PRLabel: %WebPubSub
-/sdk/web-pubsub/ @vicancy @xirzec @bterlson
+# ServiceLabel: %WebPubSub
+/sdk/web-pubsub/ @vicancy
 
 # PRLabel: %EngSys
+# ServiceLabel: %EngSys
+# AzureSdkOwners: @ckairen @mikeharder @weshaggard @benbp
 /sdk/template/ @ckairen @mikeharder @weshaggard @benbp
 
+# PRLabel: %test-utils
+# ServiceLabel: %test-utils
+# AzureSdkOwners: @HarshaNalluru @timovv
 /sdk/test-utils/ @HarshaNalluru @timovv
 
 # PRLabel: %Cognitive - Text Analytics
-/sdk/textanalytics/ @quentinRobinson @wangyuantao @minhanh-phan @deyaaeldeen @witemple-msft
+# ServiceLabel: %Cognitive - Text Analytics
+/sdk/textanalytics/ @quentinRobinson
 
 # PRLabel: %Cognitive - Language
-/sdk/cognitivelanguage/ @quentinRobinson @wangyuantao @minhanh-phan @deyaaeldeen
+# ServiceLabel: %Cognitive - Language
+/sdk/cognitivelanguage/ @quentinRobinson
+
+# ServiceLabel: %OpenAI
+# ServiceOwners: @glharper
 
 # PRLabel: %OpenAI
 /sdk/openai/ @glharper @deyaaeldeen @minhanh-phan
 
 # PRLabel: %Image Analysis
-/sdk/vision/ai-vision-image-analysis-rest/ @rhurey @dargilco
-
-# ServiceLabel: %Image Analysis %Service Attention
-#/<NotInRepo>/ @rhurey @dargilco
+# ServiceLabel: %Image Analysis
+/sdk/vision/ai-vision-image-analysis-rest/ @rhurey @dargilco 
 
 # PRLabel: %Schema Registry
-/sdk/schemaregistry/ @deyaaeldeen
+# ServiceLabel: %Schema Registry
+# AzureSdkOwners: @deyaaeldeen @minhanh-phan
+/sdk/schemaregistry/ @deyaaeldeen @minhanh-phan
 
 # PRLabel: %Cognitive - Form Recognizer
-/sdk/formrecognizer/ @witemple-msft @HarshaNalluru @jeremymeng
+# ServiceLabel: %Cognitive - Form Recognizer
+# AzureSdkOwners: @HarshaNalluru @jeremymeng
+/sdk/formrecognizer/ @HarshaNalluru @jeremymeng
 
 # PRLabel: %Cognitive - Metrics Advisor
+# ServiceLabel: %Cognitive - Metrics Advisor
+# AzureSdkOwners: @KarishmaGhiya @jeremymeng
 /sdk/metricsadvisor/ @KarishmaGhiya @jeremymeng
 
 # PRLabel: %Cognitive - Content Safety
-/sdk/contentsafety/ @bowgong @mengai @minjuewu
+# ServiceLabel: %Cognitive - Content Safety
+# AzureSdkOwners: @xirzec
+/sdk/contentsafety/ @xirzec
```

Comment:
As this is service for DPG, shall we let service team to be the code owner?

Reply:
Yes, the problem is that all three of them are invalid owners as far as GitHub is concerned:

<img width="451" alt="image" src="https://github.com/Azure/azure-sdk-for-js/assets/639216/a32a0a3b-52bd-48b5-b6e8-4d8a623447b9">

Perhaps we could make them label owners but not path owners?